rust/relay/server/src/lib.rs

```diff
@@ -161,3 +161,20 @@ impl fmt::Display for PeerSocket {
         self.0.fmt(f)
     }
 }
+
+#[derive(clap::ValueEnum, Debug, Clone, Copy)]
+pub enum EbpfAttachMode {
```

Comment:
Nit: This could go into the `ebpf` module and just be called `AttachMode`. Would be more cohesive.